docs/source/contributing.rst

```diff
@@ -72,12 +64,14 @@ Writing a new dataset loader
 #############################
 
 
-The steps to add a new dataset loader to ``mirdata`` are:
+The steps to add a new dataset loader to Mirdata are:
 
 1. `Create an index <create_index_>`_
 2. `Create a module <create_module_>`_
 3. `Add tests <add_tests_>`_
-4. `Submit your loader <submit_loader_>`_
+4. `Update Mirdata documentation <update_docs_>`_
+5. `Upload index to Zenodo <upload_index_>`_
+6. `Create a Pull Request on GitHub <create_pr_>`_
 
 
 Before starting, check if your dataset falls into one of these non-standard cases:
```

Comment:
I think the last case in the non-standard cases is a little confusing as it only suggests uploading the index if size > 5MB

Is the index large (e.g. > 5 MB)? If so, see [this section](https://mirdata--641.org.readthedocs.build/en/641/source/contributing.html#large-index)